src/analysis/fourier/add_circle.lean

```diff
@@ -326,4 +348,75 @@ begin
   ring,
 end
 
+/-- The Fourier coefficients of a continuous function are given by integrating over the circle. This
+is a shortcut to avoid messing around with ae_equivalence classes. -/
+lemma fourier_series_repr_continuous (f : C(add_circle T, ℂ)) (n : ℤ) :
+  fourier_series.repr (to_Lp 2 haar_add_circle ℂ f) n =
+  ∫ (t : add_circle T), fourier (-n) t * f t ∂haar_add_circle :=
+begin
+  rw fourier_series_repr,
+  apply integral_congr_ae,
+  refine filter.eventually_eq.mul (filter.eventually_of_forall (by tauto)) _,
+  convert continuous_map.coe_fn_to_ae_eq_fun haar_add_circle f using 1,
+end
+
+/-- The Fourier coefficients of a continuous function are given by integrating over the interval
+`[a, a + T] ⊂ ℝ`, for any `a`. -/
+lemma fourier_series_repr_continuous' (f : C(add_circle T, ℂ)) (n : ℤ) (a : ℝ) : fourier_series.repr
+(to_Lp 2 haar_add_circle ℂ f) n = 1 / T * ∫ x in a .. a + T, @fourier T (-n) x * f x :=
```

Comment:
```suggestion
lemma fourier_series_repr_continuous' (f : C(add_circle T, ℂ)) (n : ℤ) (a : ℝ) : 
  fourier_series.repr (to_Lp 2 haar_add_circle ℂ f) n 
    = 1 / T * ∫ x in a .. a + T, @fourier T (-n) x * f x :=
```
indent lemma statement by two spaces. Also, try to break the formulas at the most meaningful places.